api/app/Http/Resources/UserResource.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+use Illuminate\Database\Eloquent\Collection;
+
+use App\Models\Department;
+use GraphQL\Experimental\Executor\Collector;
+
+class UserResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array|\Illuminate\Contracts\Support\Arrayable|\JsonSerializable
+     */
+    public function toArray($request)
+    {
+
+        $awardExperiences = AwardExperienceResource::collection($this->awardExperiences);
+        $communityExperiences = CommunityExperienceResource::collection($this->communityExperiences);
+        $educationExperiences = EducationExperienceResource::collection($this->educationExperiences);
+        $personalExperiences = PersonalExperienceResource::collection($this->personalExperiences);
+        $workExperiences = WorkExperienceResource::collection($this->workExperiences);
+
+        $collection = collect();
+        $collection = $collection->merge($awardExperiences);
+        $collection = $collection->merge($communityExperiences);
+        $collection = $collection->merge($educationExperiences);
+        $collection = $collection->merge($personalExperiences);
+        $collection = $collection->merge($workExperiences);
+
+        return [
+            'id' => $this->id,
+            'sub' => $this->sub,
+            'roles' => $this->roles,
+            'firstName' => $this->firstName,
+            'lastName' => $this->lastName,
```

Comment:
this too please

Reply:
Updated name as well: [update user resource name attribute case](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3938/commits/843f4b75deebc223252a17e2d3c28f44c3edb46d)

Question, is there a reason we are case changing from `snake_case` in DB columns to `camelCase` in the `JSON`?

Reply:
good question, let's use the phone a friend lifeline
@petertgiles

Reply:
I think it's just a convention thing.  Convention in php/laravel is snake_case and JSON is cameCase?